docs/content/docs/deployment/network_buffer.md

```diff
@@ -0,0 +1,125 @@
+---
+title: "Network Tuning"
+weight: 100
+type: docs
+aliases:
+  - /deployment/network_buffer.html
```

Comment:
rename the file to `network_mem_tuning.md`? (to be consistent with `mem_tuning.md`).